invenio_rdm_records/config.py

```diff
@@ -356,6 +357,34 @@ def always_valid(identifier):
     }
 """
 
+RDM_PARENT_PERSISTENT_IDENTIFIER_PROVIDERS = [
+    # DataCite Concept DOI provider
+    providers.DataCitePIDProvider(
+        "datacite",
+        client=providers.DataCiteClient(
+            "datacite",
+            config_prefix="DATACITE",
+        ),
+        # TODO: Figure out how to conditionally serialize for Concept DOI
+        # serializer=DataCite43JSONSerializer(schema_context={
+        #     "doi_identifier_from": "parent.pids"
+        # }),
```

Comment:
shelve: DataCite serialization needs a bit of imagination to get it working without making too many changes in the existing schemas. It can be addressed separately though, since for now it would just fail or be wrong.

Reply:
https://github.com/inveniosoftware/invenio-rdm-records/issues/1318